packages/edit-site/src/components/global-styles/screen-revisions/get-revision-changes.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+const translationMap = {
+	caption: __( 'caption' ),
+	link: __( 'link' ),
+	button: __( 'button' ),
+	heading: __( 'heading' ),
+	'settings.color': __( 'color settings' ),
+	'settings.typography': __( 'typography settings' ),
+	'color.text': __( 'text colors' ),
+	'color.background': __( 'background colors' ),
+	'spacing.margin': __( 'margin styles' ),
+	'spacing.padding': __( 'padding styles' ),
+	'spacing.blockGap': __( 'block gap' ),
+	'settings.layout': __( 'layout settings' ),
```

Comment:
From a user perspective, the only things that will have changed here is content or wide size, I think? So `settings` might not feel very direct to the user. We could either split this out into content/wide size, or use `layout styles` for consistency?

Another idea, though, could be to remove the word `style` or `styles` from each of these strings, and use `style` or plural `styles` at the end of the text string?